TelemetrySite/server/data_upload_scripts/data_upload.py

```diff
@@ -84,17 +109,10 @@ def upload_data_in_chunks(collection_access_event, object_id, data_values_json):
                     break
                 data.append(sliced_data[data_index+1].pop())
                 
-        data_to_insert = {"messages":data}
+        data_to_insert = {"context":new_run_data, "messages":data}
```

Comment:
Data is being uploaded to an independent field called "messages" when it should be uploaded to "context.event.runs[0].messages"
```suggestion
        data_to_insert = {"context":new_run_data, "context.event.runs[0].messages":data}
```